types/dinero.js/index.d.ts

```diff
@@ -1,8 +1,9 @@
-// Type definitions for dinero.js 1.4
+// Type definitions for dinero.js 1.6
 // Project: https://sarahdayan.github.io/dinero.js
 // Definitions by: BendingBender <https://github.com/BendingBender>
+//                 David Acosta <https://github.com/juandaco>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.1
+// TypeScript Version: 3.4
```

Comment:
Please set this to the minimum version of TypeScript that is compatible with these type definitions.
Now that you don't use the `readonly` keyword, this should be much lower than 3.4.